justfile

```diff
@@ -1,21 +1,24 @@
 # Use values from .env automatically, if present
-set dotenv-load
+
+set dotenv-load := true
 
 alias r := run
-alias b  := build
+alias b := build
 alias bb := bbuild
 
 # example: platform := "--platform=linux/amd64"
+
 platform := ""
-image := "rnacentral/r2dt"
+platform_arg := if platform == "" { "" } else { "--platform=" + platform }
 base_image := "rnacentral/r2dt-base"
+image := "rnacentral/r2dt"
 tag := "latest"
```

Comment:
@anayden The `tag` variable does not seem to be used in the script, so it should be either removed (👎) or used in combination with `BASE_IMAGE_VERSION` argument (👍).

Reply:
It's unused indeed, and I don't think it's needed in combination with `BASE_IMAGE_VERSION`: when `just full-build` is executed we specifically want `latest` to be hardcoded as the value for `BASE_IMAGE_VERSION` because that's the tag base image will be locally built with. Any changes to this value will actually break `full-build`.

So, I'm sorry but `tag` has to go :(